DESCRIPTION

```diff
@@ -1,14 +1,16 @@
 Package: QFASA
 Title: Quantitative Fatty Acid Signature Analysis
 Version: 1.0.3
+Date: 2018-08-14
 Authors@R: c(
              person("Sara", "Iverson", email = "Sara.Iverson@dal.ca", role = c("aut")),
              person("Chris", "Field", email = "field@mathstat.dal.ca", role = c("aut")),
```

Comment:
@cstewartGH is this the correct spelling for Chris Field ? I see there was a bug to fix the spelling of his name. I see it was Fields in the citiation.